.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug Report
+description: 'Use this template for tracking new bugs. Make sure to add the severity in the issue title (Low, Medium or High)title: "BUG: [SEVERITY]: [BUG DESCRIPTION]"'
```

Comment:
I'm wondering if it's a good idea to allow users to assign the severity themselves, in a situation where many people are opening bugs the severity tag would likely loose meaning. Maybe we can assign that ourselves as the bugs are reviewed?

Reply:
thats a good point.. I'll remove the severity and we can be the ones who assign it after reviewing like you said!